test/core/util/port_isolated_runtime_environment.cc

```diff
@@ -67,6 +67,10 @@ int grpc_pick_unused_port_or_die(void) {
   }
 }
 
-void grpc_recycle_unused_port(int port) { (void)port; }
+static void isolated_recycle_unused_port(int port) { (void)port; }
+
+const auto grpc_isolated_port_picker_prev_fns =
```

Comment:
Looks like you intended to use this later, but I don't see it. And the fuzzing EE is not yet overriding the picker functions. Did you decide the override was unnecessary?

Reply:
tricking the linker into not stripping this code... let's see if it's still necessary after other things are fixed.

Reply:
fuzzing event engine is setting them, in the constructor (line 156 fuzzing_event_engine.cc in the version in my editor)